be/src/main/java/com/jjikmuk/sikdorak/review/repository/ReviewRepository.java

```diff
@@ -69,4 +69,14 @@ SELECT avg(r.reviewScore.reviewScore) FROM Review r
          WHERE r.storeId = :storeId AND r.deleted = false""")
     Optional<Double> findReviewScoreAverageByStoreId(@Param("storeId") long storeId);
 
+    @Query("""
+        SELECT r FROM Review r
+         WHERE r.storeId = :storeId
+           AND r.id <= :targetId
+         ORDER BY r.id DESC
+        """)
```

Comment:
p1; 쿼리가 public, protected, private 모두 조회될 거 같은데요!?

이야기 나눴던, public만 조회하는건 어떠신가요?

```suggestion
    @Query("""
        SELECT r FROM Review r
         WHERE r.storeId = :storeId
           AND r.reviewVisibility = 'PUBLIC'
           AND r.id <= :targetId
         ORDER BY r.id DESC
        """)
```

Reply:
추가한다는게 깜빡했네요;; 수정해서 push 했습니다